cloud/jenkins/psmdb_operator_gke_latest.groovy

```diff
@@ -1,79 +1,100 @@
-GKERegion='us-central1-a'
+region='us-central1-a'
 tests=[]
 clusters=[]
 
-void createCluster(String CLUSTER_SUFFIX) {
-    clusters.add("${CLUSTER_SUFFIX}")
-    if ( "${params.CLUSTER_WIDE}" == "YES" ) {
-        env.OPERATOR_NS = 'psmdb-operator'
-    }
-
-    withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-alpha-key-file', variable: 'CLIENT_SECRET_FILE')]) {
-        sh """
-            export KUBECONFIG=/tmp/$CLUSTER_NAME-${CLUSTER_SUFFIX}
-            export USE_GKE_GCLOUD_AUTH_PLUGIN=True
-            source $HOME/google-cloud-sdk/path.bash.inc
-            ret_num=0
-            while [ \${ret_num} -lt 15 ]; do
-                ret_val=0
-                gcloud auth activate-service-account --key-file $CLIENT_SECRET_FILE && \
-                gcloud config set project $GCP_PROJECT && \
-                gcloud alpha container clusters create --release-channel rapid $CLUSTER_NAME-${CLUSTER_SUFFIX} --zone $GKERegion --cluster-version $PLATFORM_VER --project $GCP_PROJECT --preemptible --disk-size 30 --machine-type n1-standard-4 --num-nodes=4 --min-nodes=4 --max-nodes=6 --network=jenkins-vpc --subnetwork=jenkins-${CLUSTER_SUFFIX} --cluster-ipv4-cidr=/21 --labels delete-cluster-after-hours=6 --enable-ip-alias --workload-pool=cloud-dev-112233.svc.id.goog && \
-                kubectl create clusterrolebinding cluster-admin-binding1 --clusterrole=cluster-admin --user=\$(gcloud config get-value core/account) || ret_val=\$?
-                if [ \${ret_val} -eq 0 ]; then break; fi
-                ret_num=\$((ret_num + 1))
-            done
-            if [ \${ret_num} -eq 15 ]; then exit 1; fi
-        """
-   }
-}
-
-void shutdownCluster(String CLUSTER_SUFFIX) {
+void prepareNode() {
+    echo "=========================[ Installing tools on the Jenkins executor ]========================="
+    sh """
+        sudo curl -s -L -o /usr/local/bin/kubectl https://dl.k8s.io/release/\$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl && sudo chmod +x /usr/local/bin/kubectl
+        kubectl version --client --output=yaml
+
+        curl -fsSL https://get.helm.sh/helm-v3.12.3-linux-amd64.tar.gz | sudo tar -C /usr/local/bin --strip-components 1 -xzf - linux-amd64/helm
+
+        sudo sh -c "curl -s -L https://github.com/mikefarah/yq/releases/download/v4.35.1/yq_linux_amd64 > /usr/local/bin/yq"
+        sudo chmod +x /usr/local/bin/yq
+
+        sudo sh -c "curl -s -L https://github.com/stedolan/jq/releases/download/jq-1.7rc2/jq-linux-amd64 > /usr/local/bin/jq"
```

Comment:
Please use the 1.6 version of jq because we use it in all other pipelines and the query language changes over versions so it's very hard to debug issues.